apps/api/src/campaign-application/campaign-application.controller.ts

```diff
@@ -1,25 +1,53 @@
-import { Controller, Get, Post, Body, Patch, Param } from '@nestjs/common'
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  ForbiddenException,
+  NotFoundException,
+  Logger,
+  Inject,
+  forwardRef,
+} from '@nestjs/common'
 import { CampaignApplicationService } from './campaign-application.service'
 import { CreateCampaignApplicationDto } from './dto/create-campaign-application.dto'
 import { UpdateCampaignApplicationDto } from './dto/update-campaign-application.dto'
 import { ApiTags } from '@nestjs/swagger'
-import { AuthenticatedUser, Public, RoleMatchingMode, Roles } from 'nest-keycloak-connect'
+import { AuthenticatedUser, RoleMatchingMode, Roles } from 'nest-keycloak-connect'
 import { RealmViewSupporters, ViewSupporters } from '@podkrepi-bg/podkrepi-types'
 import { KeycloakTokenParsed, isAdmin } from '../auth/keycloak'
+import { PersonService } from '../person/person.service'
 
 @ApiTags('campaign-application')
 @Controller('campaign-application')
 export class CampaignApplicationController {
-  constructor(private readonly campaignApplicationService: CampaignApplicationService) {}
+  constructor(
+    private readonly campaignApplicationService: CampaignApplicationService,
+    @Inject(forwardRef(() => PersonService)) private readonly personService: PersonService,
```

Comment:
`@Inject` won't be needed if we import the PersonModule, to the `CampaignApplicationModule`. Please refer to one of the other comments from more information.